src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -591,7 +591,7 @@ protected override SupervisorStrategy SupervisorStrategy()
                         {
                             var causedBy = ia.InnerException == null
                                 ? ""
-                                : string.Format("Caused by: [{0}]", ia.InnerException);
+                                : $"Caused by: [{ia.InnerException}]";
```

Comment:
It seems that the concept of the known issue was here before. If there is inner exception, we print the message and the stack trace. No changes here.